recipes/libnvtx/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "3.0.1" %}
+{% set sha256 = "bb8d1536aad708ec807bc675e12e5838c2f84481dec4005cd7a9bbd49e326ba1" %}
+
+package:
+  name: libnvtx
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/NVTX/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: mkdir -p "$PREFIX"/include && cp -r c/include/nvtx3 "$PREFIX"/include  # [not win]
```

Comment:
This should be in a `build.sh`

Reply:
Thanks! Fixed.

Apologies, but this PR is not ready to merge yet as we are still giving some thought to the package name. I converted this PR to a draft.